src/app/feed/feed-create-post/feed-create-post.component.ts

```diff
@@ -86,6 +86,12 @@ export class FeedCreatePostComponent implements OnInit {
     }
   }
 
+  uploadFile(event: any): void {
+    console.log("Dropped Image");
+
+    this._handleFileInput(event[0]);
```

Comment:
we should skip uploading if `isComment` is true to reduce image uploads that aren't going to be displayed

Reply:
lololol....you know I want comments so bad hah, will do! lol